test/suites/optional/multus.robot

```diff
@@ -0,0 +1,151 @@
+${PREEXISTING_BRIDGE_POD_NAME}      test-bridge-preexisting
+${PREEXISTING_BRIDGE_IP}            10.10.1.10/24
+
+${NS}                               default
+
+
+*** Test Cases ***
+Pre-Existing Bridge Interface
+    [Documentation]    Test verifies if Bridge CNI plugin will work correctly with pre-existing interface.
+    [Setup]    Run Keywords
+    ...    Interface Should Not Exist    ${PREEXISTING_BRIDGE_INTERFACE}
+    ...    AND
+    ...    Create Interface    ${PREEXISTING_BRIDGE_INTERFACE}
+    ...    AND
+    ...    Create NAD And Pod    ${PREEXISTING_BRIDGE_NAD_YAML}    ${PREEXISTING_BRIDGE_POD_YAML}
+    ...    AND
+    ...    Named Pod Should Be Ready    ${PREEXISTING_BRIDGE_POD_NAME}    ${NS}
+    ...    AND
+    ...    Interface Should Exist    ${PREEXISTING_BRIDGE_INTERFACE}
+    ...    AND
+    ...    Set IP For Host Interface    ${PREEXISTING_BRIDGE_INTERFACE}    ${PREEXISTING_BRIDGE_IP}
+
+    Curl Pod Over Local Interface    ${PREEXISTING_BRIDGE_POD_NAME}    ${NS}    ${PREEXISTING_BRIDGE_INTERFACE}
+
+    [Teardown]    Cleanup Bridge Test
+    ...    ${PREEXISTING_BRIDGE_NAD_YAML}
+    ...    ${PREEXISTING_BRIDGE_POD_YAML}
+    ...    ${PREEXISTING_BRIDGE_INTERFACE}
+
+No Pre-Existing Bridge Interface
+    [Documentation]    Test verifies if Bridge CNI plugin will
+    ...    work correctly if there is no pre-existing bridge
+    ...    interface - it needs to be created.
+    [Setup]    Run Keywords
+    ...    Interface Should Not Exist    ${BRIDGE_INTERFACE}
+    ...    AND
+    ...    Create NAD And Pod    ${BRIDGE_NAD_YAML}    ${BRIDGE_POD_YAML}
+    ...    AND
+    ...    Named Pod Should Be Ready    ${BRIDGE_POD_NAME}    ${NS}
+    ...    AND
+    ...    Interface Should Exist    ${BRIDGE_INTERFACE}
+    ...    AND
+    ...    Set IP For Host Interface    ${BRIDGE_INTERFACE}    ${BRIDGE_IP}
+
+    Curl Pod Over Local Interface    ${BRIDGE_POD_NAME}    ${NS}    ${BRIDGE_INTERFACE}
+
+    [Teardown]    Cleanup Bridge Test
+    ...    ${BRIDGE_NAD_YAML}
+    ...    ${BRIDGE_POD_YAML}
+    ...    ${BRIDGE_INTERFACE}
+
+
+*** Keywords ***
+Setup
+    [Documentation]    Test suite setup
+    Check Required Env Variables
+    Login MicroShift Host
+    Setup Kubeconfig
+
+Teardown
+    [Documentation]    Test suite teardown
+    Logout MicroShift Host
+    Remove Kubeconfig
+
+Create NAD And Pod
+    [Documentation]    Creates provided NetworkAttachmentDefinition and Pod.
+    [Arguments]    ${nad}    ${pod}
+    Oc Create    -f ${nad}
+    Oc Create    -f ${pod}
+
+Cleanup Bridge Test
+    [Documentation]    Removes provided NetworkAttachmentDefinition, Pod and network interface to allow for test rerun.
+    [Arguments]    ${nad}    ${pod}    ${if}
+    Run Keyword And Continue On Failure
+    ...    Oc Delete    -f ${pod}
+    Run Keyword And Continue On Failure
+    ...    Oc Delete    -f ${nad}
+    Command Should Work    ip link delete ${if}
+
+Curl Pod Over Local Interface
```

Comment:
Nit: Curl is an implementation detail. Should we call it `Connect Pod` or something?

Reply:
`Connect To Pod Over Local Interface`? Because in future PRs there's `Curl Pod From The Hypervisor` so we need to be more specific